README.md

```diff
@@ -1,10 +1,8 @@
 # ACPaaS UI - React Components
 
-**This documentation is work-in-progress!**
-
 ## Introduction
 
-Antwerp City Platform as a Service User Interface (ACPaas UI) is a **component interface library** for building user interfaces and responsive web apps. It's designed to provide developers with functionality and UI/UX patterns that matches the Antwerpen styleguide.
+Antwerp City Platform as a Service User Interface (ACPaas UI) is a **component interface library** for building user interfaces and responsive web apps. It's designed to provide developers with functionality en UI/UX patterns that matches the Antwerpen styleguide.
```

Comment:
Typo: `and` instead of `en`.

Reply:
Hmm, copy/paste vanop angular repo gedaan. Zal het daar ook meteen aanpassen.